README.md

```diff
@@ -13,80 +13,73 @@
+## Releases
 
 We abide by [semantic versioning](https://semver.org/). Public APIs can change
 at any time until we reach version 1.0. We will, however, bump the minor version
 when a backwards incompatible change is made.
 
  [v0.5](https://github.com/vmware/concord-bft/releases/tag/v0.5)
 
-Contributing
-----
-
-The concord-bft project team welcomes contributions from the community. If you wish to contribute code and you have not
-signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
-questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq). For more detailed information,
-refer to [CONTRIBUTING.md](CONTRIBUTING.md).
-
-Community
-----
-
-[Concord-BFT Slack](https://concordbft.slack.com/).
-
-Get Slack invitation via this [link](https://join.slack.com/t/concordbft/shared_invite/enQtNTAyMzg0MTk0NzU0LWViMjA4ODQ4OTcxYmVkMWU4NGYyNTZmZDQ0ZTIzZTNiZmIyZmI5Mjg0NDZjNjBkYmI2Yjg1Mzg1NGUyNjQ0OWQ) or send request to <concordbft@gmail.com>.
-
+## Build (Ubuntu Linux 18.04)
 
+We use Conan Package Manager to install all concord-bft dependencies.
+Dependencies that currently are not supported by the conan center, have custom conan installer in concord-bft/.conan
 
-Directory structure
-----
 
-- [bftengine](./bftengine): concord-bft codebase
-	- [include](./bftengine/include): external interfaces of concord-bft (to be used by client applications)
-	- [src](./bftengine/src): internal implementation of concord-bft
-    - [tests](./bftengine/tests): tests and usage examples
-- [threshsign](./threshsign): crypto library that supports digital threshold signatures
-	- [include](./threshsign/include): external interfaces of threshsign (to be used by client applications)
-	- [src](./threshsign/src): internal implementation of threshsign
-    - [tests](./threshsign/tests): tests and usage examples
-- [scripts](./scripts): build scripts
-- [tests](./tests): BFT engine system tests
+### Build concord-bft
 
-License
-----
+```sh
+git clone https://github.com/vmware/concord-bft
+cd concord-bft
+./install.sh 	# Install all dependencies 
 
-concord-bft is available under the [Apache 2 license](LICENSE).
+mkdir -p build
+cd build
 
+conan install --build missing ..
+cmake ..
+make
 
+# run a very basic test
+./tests/simpleTest/scripts/testReplicasAndClient.sh
+```
 
-Build (Ubuntu Linux 18.04)
-----
-We use Conan Package Manager to install all concord-bft dependencies.
-Dependencies that currently are not supported by the conan center, have custom conan installer in concord-bft/.conan
+### Build Options
 
-### Install Dependencies
-Install all dependencies using
+In order to turn on or off various options, you need to change your cmake configuration. This is
+done by passing arguments to cmake with a `-D` prefix: e.g. `cmake -DBUILD_TESTING=OFF`. Note that
+make must be run afterwards to build according to the configuration. The following options are
+useful for building concord-bft:
```

Comment:
I'd rather say: "The following options are available:"

Reply:
again not my text, the diff got confused.